al5d_gazebo/launch/final.launch

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0"?>
+<launch>
+
+  <!-- BASIC ENVIRONMENT -->
+  <include file="$(find gazebo_ros)/launch/empty_world.launch">
+    <arg name="world_name" value="$(find al5d_gazebo)/worlds/final.world"/>
+  </include>
+
+  <!-- OBSTACLES / OBJECT PLATFORMS -->
```

Comment:
If these lines are unused we should delete them.

Reply:
Thanks for pointing this out, done